### DISCUSSION TITLE
FreeBSD Compatibility

### DISCUSSION THREAD

**Discussion by Espionage724** [ppy/osu#29887] [Ideas] [answered]
I tried `dotnet run --project osu.Desktop` and got an error that led to something about FreeBSD not being directly-supported by something.

Wine on FreeBSD 14.1 (as I understand, at least with `devel`) is either 32-bit or 64-bit only; adding 32-bit support removing 64-bit support. I went with 32-bit Wine support, and osu! non-lazer works great (W10 `win32` prefix, `dotnet45`, but might need more for extra fonts).

I'd like to be able to build lazer native on FreeBSD, and I'm wondering if there's any obvious show-stoppers that might prevent that? (I forget the specific issue with `dotnet run`)

**Comment by Stoppedpuma** [comment, marked as answer]
Regarding a native build, the current issue at the moment seems to be `libbass`. There's a similar discussion to this at #22278, you'll likely find what you're looking for there.
